### PR TITLE
finished the task

### PR DESCRIPTION
**Pull request by EvhenIvchenko**
- [DEMO LINK](https://EvhenIvchenko.github.io/layout_calendar/)
- [TEST REPORT LINK](https://EvhenIvchenko.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/utils/functions.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@function getLimitSize($day-box-size , $boxGap) {
+  @return $day-box-size *7 + $boxGap * 6;
```
Comment (OlEzhA0) [inline src/styles/utils/functions.scss]:
```suggestion
  @return $day-box-size * 7 + $boxGap * 6;
```

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+    &--month-length--#{$lastDay} :nth-child(n + #{$lastDay+1}) {
+      display: none;
+    }
+  }
+
+  &__box {
+    position: relative;
+    width: $day-box-size;
+    height: $day-box-size;
+    box-sizing: border-box;
```
Comment (OlEzhA0) [inline src/styles/main.scss:33]: move `box-sizing: border-box;`

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,56 @@
...
+        transform: translate(50% , -50%);
+        display: block;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+      }
+    }
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      transition: all 0.5s;
```
Comment (OlEzhA0) [inline src/styles/main.scss]: move transition to &__box to prevent jump after unhover

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
hey! need some improvements

File: src/styles/main.scss (hunk 2 above)
Comment (EvhenIvchenko) [reply src/styles/main.scss:33]: move where , sorry but i didn't understand

Comment (EvhenIvchenko) [reply src/styles/main.scss:33]: Each day is a grey (#eee) 100px square (including 1px black border)

According to checklist  we need border box here.

File: src/styles/utils/functions.scss (hunk 1 above)
Comment (EvhenIvchenko) [reply src/styles/utils/functions.scss]: fixed

File: src/styles/main.scss (hunk 3 above)
Comment (EvhenIvchenko) [reply src/styles/main.scss]: fixed

File: src/styles/main.scss (hunk 2 above)
Comment (IvanFesenko) [reply src/styles/main.scss:33]: `box-sizing: border-box;` should be inside `*` selector

**Review by IvanFesenko — APPROVED** [review]

File: src/styles/main.scss (hunk 2 above)
Comment (EvhenIvchenko) [reply src/styles/main.scss:33]: okay , got it , thank you

**Pull request closed without merging**